opentracing-spring-cloud-starter-jaeger/src/main/java/io/opentracing/contrib/spring/cloud/starter/jaeger/JaegerConfigurationProperties.java

```diff
@@ -0,0 +1,286 @@
+   */
+  private boolean enableB3Propagation = false;
+  /**
+   * Enable the collections of metrics of collection process Setting this to true will enable {@link
+   * com.uber.jaeger.metrics.InMemoryStatsReporter}
+   */
+  private boolean enableMetrics = false;
+
+  public boolean isEnabled() {
+    return enabled;
+  }
+
+  public void setEnabled(boolean enabled) {
+    this.enabled = enabled;
+  }
+
+  public String getServiceName() {
+    return serviceName;
+  }
+
+  public void setServiceName(String serviceName) {
+    this.serviceName = serviceName;
+  }
+
+  public boolean isLogSpans() {
+    return logSpans;
+  }
+
+  public void setLogSpans(boolean logSpans) {
+    this.logSpans = logSpans;
+  }
+
+  public boolean isEnableB3Propagation() {
+    return enableB3Propagation;
+  }
+
+  public void setEnableB3Propagation(boolean enableB3Propagation) {
+    this.enableB3Propagation = enableB3Propagation;
+  }
+
+  public boolean isEnableMetrics() {
+    return enableMetrics;
+  }
+
+  public void setEnableMetrics(boolean enableMetrics) {
+    this.enableMetrics = enableMetrics;
+  }
+
+  public HttpSender getHttpSender() {
+    return httpSender;
+  }
+
+  public RemoteReporterProperties getRemoteReporterProperties() {
+    return remoteReporterProperties;
+  }
+
+  public UdpSender getUdpSender() {
+    return udpSender;
+  }
+
+  public ConstSampler getConstSampler() {
+    return constSampler;
+  }
+
+  public ProbabilisticSampler getProbabilisticSampler() {
+    return probabilisticSampler;
+  }
+
+  public RateLimitingSampler getRateLimitingSampler() {
+    return rateLimitingSampler;
+  }
+
+  public RemoteControlledSampler getRemoteControlledSampler() {
+    return remoteControlledSampler;
+  }
+
+
+  public static class RemoteReporterProperties {
+
+    private Integer flushInterval;
+
+    private Integer maxQueueSize;
+
+    public Integer getFlushInterval() {
+      return flushInterval;
+    }
+
+    public void setFlushInterval(Integer flushInterval) {
+      this.flushInterval = flushInterval;
+    }
+
+    public Integer getMaxQueueSize() {
+      return maxQueueSize;
+    }
+
+    public void setMaxQueueSize(Integer maxQueueSize) {
+      this.maxQueueSize = maxQueueSize;
+    }
+  }
+
+  public static class HttpSender {
+
+    private String url;
+
+    private Integer maxPayload;
+
+    private boolean disable = false;
+
+    public String getUrl() {
+      return url;
+    }
+
+    public void setUrl(String url) {
+      this.url = url;
+    }
+
+    public Integer getMaxPayload() {
+      return maxPayload;
+    }
+
+    public void setMaxPayload(Integer maxPayload) {
+      this.maxPayload = maxPayload;
+    }
+
+    public boolean isDisable() {
+      return disable;
+    }
+
+    public void setDisable(boolean disable) {
+      this.disable = disable;
+    }
+  }
+
+  public static class UdpSender {
+
+    private String host;
+
+    private int port;
+
+    private int maxPacketSize = 65000;
```

Comment:
There doesn't seem to be a builder that would allow us to set only some of the properties, so using the "full" is the easiest option.
It might make sense however to have default value of `0`

Reply:
Yes, then use 0, it's also mentioned in javadoc.

Reply:
Done